sfaira/unit_tests/update_loaders/update_celltypes.py

```diff
@@ -10,7 +10,7 @@
 
 SYMBOL_COL = "target"
 ID_COL = "target_id"
-UNKNOWN_IDS = ["UNKNOWN", "NOT_A_CELL"]
+unknown_IDS = ["unknown", "NOT_A_CELL"]
```

Comment:
This list is a global constant and should be upper case I think?

Reply:
indeed, will fix!